scripts/install_third_party_test.py

```diff
@@ -431,20 +449,15 @@ def test_install_redis_cli_function_calls(self):
             'download_and_untar_files_is_called': True
         }
         def mock_download_and_untar_files(
-                unused_source_url, unused_target_parent_dir,
-                unused_tar_root_name, unused_target_root_name):
+            unused_source_url: str,
+            unused_target_parent_dir: str,
+            unused_tar_root_name: str,
+            unused_target_root_name: str
+        ) -> None:
             check_function_calls['download_and_untar_files_is_called'] = True
 
-        def mock_call(unused_cmd_tokens, *args, **kwargs):  # pylint: disable=unused-argument
+        def mock_call(unused_cmd_tokens: List[str], **kwargs: int) -> Ret:  # pylint: disable=unused-argument
```

Comment:
Why here `**kwargs` is int? I think we should use str for arbitrary arguments.

Reply:
done